packages/opentelemetry-metrics/src/Metric.ts

```diff
@@ -133,7 +133,7 @@ export class CounterMetric extends Metric<BoundCounter> implements api.Counter {
    * @param labels key-values pairs that are associated with a specific metric
    *     that you want to record.
    */
-  add(value: number, labels: api.Labels) {
+  add(value: number, labels: api.Labels = {}) {
```

Comment:
This is nice, it would be nice if you can do the same thing to `measure.record` (https://github.com/astorm/opentelemetry-js/blob/astorm/labels-optional/packages/opentelemetry-metrics/src/Metric.ts#L166)

Reply:
Just realized, Prometheus exporter would fail after this change. Especially below line due to `toString` method on `undefined`:

https://github.com/open-telemetry/opentelemetry-js/blob/c454d6cf2c895be39e7ae9d1b55c039ed952dcb4/packages/opentelemetry-exporter-prometheus/src/prometheus.ts#L187

I would prefer to fix it in this same PR and do some end to end manual testing. WDYT?

Reply:
Also, in order to pass the build you need to make the same change in the API package.

Reply:
Thanks for the guidance and patience @mayurkale22 -- I'm new to the repo and the project and I'm still getting my sea legs.  

Fixing this for the `MeasureMetric` in this PR makes sense.  Fixing the prometheus exporter in this PR also makes sense.  I presume the change in `opentelemetry-api` would be to the interfaces, and that also makes sense to do in this PR. 

I'll ping -- you? -- via a PR comment when that's done.  If there's more to The Process™ please let me know. 

Best wishes.

Reply:
SGTM, just add a comment on this thread when new changes are ready to review. I usually use [this example](https://github.com/open-telemetry/opentelemetry-js/tree/master/examples/prometheus) to test against  prom. exporter.

Reply:
@mayurkale22 not sure why you think the prometheus exporter would fail on that line. Can you explain? I'm hesitant to approve changes to the `_registerMetric` routine in the prometheus exporter because the prom-client has some surprising behavior, and that is the logic we use to destroy and recreate the counters with the new values.

Reply:
> @mayurkale22 not sure why you think the prometheus exporter would fail on that line. Can you explain?

Looks like @astorm already mention the reason, here-> https://github.com/open-telemetry/opentelemetry-js/pull/1032#discussion_r421580110 (due to `toString` method on `undefined` label value).

> I'm hesitant to approve changes to the _registerMetric routine in the prometheus exporter because the prom-client ....

I would suggest to add new test in prom. exporter with some `LabelKeys` but w/o `Labels` (values). You can reuse existing test ([like this one](https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-exporter-prometheus/test/prometheus.test.ts#L200)), just dont bind the labels to metric.